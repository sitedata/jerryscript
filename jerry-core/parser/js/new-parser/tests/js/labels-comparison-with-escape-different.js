\u006ca\u0062elx: \u006cab\u0065ly: expr;
