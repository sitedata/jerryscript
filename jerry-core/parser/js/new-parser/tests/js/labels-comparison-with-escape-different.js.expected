
--- Script parsing start ---

  [  1] CBC_PUSH_LITERAL idx:0->ident(expr)
  [  0] CBC_POP_BLOCK

Final byte code dump:

  Maximum stack depth: 1
  Flags: [small_lit_enc]
  Argument range end: 0
  Register range end: 0
  Identifier range end: 1
  Const literal range end: 1
  Literal range end: 1

   0 : CBC_PUSH_LITERAL idx:0(ident)->ident(expr)
   2 : CBC_POP_BLOCK
   3 : CBC_RETURN_WITH_BLOCK

Byte code size: 4 bytes

Script parsing successfully completed. Total byte code size: 4 bytes

--- Script parsing end ---

